ingest/src/commands/all.test.ts

```diff
@@ -1,9 +1,7 @@
 import { PageStore, EmbeddedContentStore, assertEnvVars } from "chat-core";
 import { MongoClient } from "mongodb";
-import "dotenv/config";
-import { INGEST_ENV_VARS } from "../IngestEnvVars";
 import { doAllCommand } from "./all";
-import { makeIngestMetaStore } from "../IngestMetaStore";
+import { INGEST_ENV_VARS, makeIngestMetaStore } from "../";
```

Comment:
should we not import from the `index.ts` files b/c it can create circular dependencies?

Reply:
Yes, rethought this greatly. Not just for circular dependencies but also as an anti-pattern. Thanks!